src/Platform.Handlers/src/Xamarin.Platform.Handlers/Primitives/ButtonContentLayout.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Diagnostics;
+
+namespace Xamarin.Forms
+{
+	[DebuggerDisplay("Image Position = {Position}, Spacing = {Spacing}")]
+	public class ButtonContentLayout
```

Comment:
It will not have to be boxed frequently etc, so, could be possible!. For now is just the same `ButtonContentLayout` class from Xamarin.Forms. Should we do it while porting code (now) or wait to have everything ported and then adapt commented changes?

Reply:
Maybe a separate PR to catch any other issues it causes.

Reply:
Created an issue for this: https://github.com/xamarin/Xamarin.Forms/issues/13816